Testscripts/Linux/STOR-Verify-Sector-Size.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# Description:
+#    This script will verify logical sector size for 512 only and physical
```

Comment:
rewrite the description to a more accurate one

Reply:
Rewrote